Uplift/Views/GymDetail/GymDetailWeekView.swift

```diff
@@ -14,7 +14,12 @@ import SnapKit
 class GymDetailWeekView: UIView {
 
     // MARK: - Public
-    var selectedDay: WeekDay?
+    var selectedDay: WeekDay = .sunday
+    weak var delegate: WeekDelegate? {
+        didSet {
+            self.delegate?.didChangeDay(day: selectedDay)
+        }
+    }
```

Comment:
why are we calling it right away?

Reply:
doing that makes the times label synced with the calendar as soon as is made. without that, it would be initialize with a some default value, then update once the user selects a new day